README.md

```diff
@@ -33,12 +33,13 @@ To get started, you will need to make sure that pixi is installed
 ([instructions here](https://pixi.sh/latest),
 we found that using the pixi provided script was best on a Mac).
 
-To create the virtual environment, run
+To create the virtual environment, navigate to the root folder of this repository and run from your shell:
 
 ```sh
 pixi install
 pixi run pre-commit install
 ```
+This will create a python environment in "{LOCALPATH}\emissions_harmonization_historical\.pixi\envs\default\", with "python.EXE" in that folder being the default python interpreter (to be used if your for instance selecting a kernel for running a jupyter notebook in VSCode or any other IDE).
```

Comment:
```suggestion

This will create a python environment in "{LOCALPATH}/emissions_harmonization_historical/.pixi/envs/default/", with a Python executable in that folder. You should set this Python interpreter as the default in your IDE (to be used if your for instance selecting a kernel for running a jupyter notebook in VSCode or any other IDE).
```

It's a bit weird to have this environment management stuff in the README (you'd have to duplicate it in every repo you worked with, it would be like including a guide to git in the README), but if you really want it, keep it (just please don't use windows paths, they look so broken).

Reply:
Yes fair, the copying of a windows path was my oversight.